src/services/data_model_engine.js

```diff
@@ -213,23 +213,19 @@ export default class DataModelEngine {
     await this.entityRepository.discardBundling(bundleStubId);
   }
 
-  async uploadAcceptedBundleCandidates() {
+  async uploadAcceptedBundleCandidates(callbacks = {success: async () => {}, fail: async () => {}}) {
```

Comment:
Suggestion: rename `callbacks` to `uploadProgressCallbacks`

Reply:
✅